src/routes/(app)/_components/layout/navbar.svelte

```diff
@@ -9,9 +18,17 @@
     </a>
   </div>
   <div class="flex justify-end gap-7">
-    <a href="#/">Equipa</a>
-    <a href="#/">Projetos</a>
-    <a href="#/">Eventos</a>
-    <a href="#/">Contactos</a>
+    <a class:active={currentUrl === "http://localhost:3000/#/team/"} href="#/team/"><p class = "font-bold">Equipa</p></a>
+    <a class:active={currentUrl === "http://localhost:3000/#/projects/"} href="#/projects/"><p class = "font-bold">Projetos</p></a>
+    <a class:active={currentUrl === "http://localhost:3000/#/events/"} href="#/events/"><p class = "font-bold">Eventos</p></a>
+    <a class:active={currentUrl === "http://localhost:3000/#/contacts/"} href="#/contacts/"><p class = "font-bold">Contactos</p></a>
```

Comment:
I also think we can avoid having this much repetition and do a more general code, here is a hint as to how I think that could be done:  
`  const links = [
    { href: '/team', label: 'Equipa' }, ...
  ];`

Reply:
DONEEEEE!!!!!